### DISCUSSION TITLE
Error with Artisan 3.0.2

### DISCUSSION THREAD

**Discussion by muexl** [artisan-roaster-scope/artisan#1739] [Q&A]
Hi,

i am using an older Artisan Version 2.8.4 on my 2019 MacBook Pro, which works pretty good.
I want to use my 2013 Mac Pro with artisan 3.0.2. The Setup works, but when I try. to load the Presets or Profiles from Dropbox I get this Error: version = 3.0.2

1 14:05:52.392 Exception: /Users/rosterei/Dropbox/Torreo/Rösten/Artisan Presets/Genio_09_2024.aset not a valid settings file.

I already tried to use the 2.8.4 Version, but the error still appears. Any recommendations what to do next?

**Comment by MAKOMO** [comment]
A computer from 2013 is quit old. Which OS version is it running? Did you install the Artisan mac legacy or regular build?

Please take that Artisan artisan-settings.aset file, rename it to end in .txt and do a drag-and-drop to a comment field here on GitHub in this thread. We will take a look.

**Reply by muexl** [reply]
Mac OS Monterey, 12.7.6.


[Genio_09_2024.txt](https://github.com/user-attachments/files/17703198/Genio_09_2024.txt)

**Reply by MAKOMO** [reply]
I cannot duplicate the issue. The settings file you attached loads without any errors on Artisan 3.0.2 (regular and legacy build) as well as the current development version.

What are the exact steps to reproduce your issue? Did you try to copy the file you plan to load from your DropBox to your Desktop and load from there. Maybe there is an issue with your DropBox configuration!?

**Reply by muexl** [reply]
Vielen Dank Marko,

ich probiere die Files mal Lokal abzuspeichern und von dort aus in Artisan zu laden. LG
